gensim/models/wrappers/ldamallet.py

```diff
@@ -245,11 +247,20 @@ def show_topic(self, topicid, topn=10):
     def print_topic(self, topicid, topn=10):
         return ' + '.join(['%.3f*%s' % v for v in self.show_topic(topicid, topn)])
 
+    #########  function to return the version of mallet   ######
+    def get_version(self, direc_path):
+        xml_path = direc_path.split("bin")[0]
+        doc = et.parse(xml_path+"pom.xml").getroot()
```

Comment:
So, in case of a jar file should I still check for version from its manifest file, by first decomposing it or not?